### DISCUSSION TITLE
Use of Zip Plugin

### DISCUSSION THREAD

**Discussion by TheRealSharky** [driftyco/ionic-native#4430] [Q&A]
If I enter in app.module.ts 

import { zip } from '@awesome-cordova-plugins/zip/ngx';
and
providers: [ ..., Zip],

I get the following error message:
Error: Invalid provider for the NgModule 'AppModule' - only instances of Provider and Type are allowed, got: [..., ?[object Object]?]

What do I need to change to use the plugin as expected?
Project: Ionic 6, Angular & Capacitor

**Comment by paulosllvn** [comment]
@danielsogl   Mate can you have a look at this? Getting the exact same a year later
